dbt_sugar/core/task/doc.py

```diff
@@ -49,27 +52,21 @@ def run(self) -> int:
         dbt_credentials = self.load_dbt_credentials()
         type_of_connection = dbt_credentials.get("type", "")
 
-        if type_of_connection == "postgres":
-            try:
-                columns_sql = PostgresConnector(
-                    user=dbt_credentials.get("user", str()),
-                    password=dbt_credentials.get("password", str()),
-                    host=dbt_credentials.get("host", str()),
-                    database=dbt_credentials.get("database", str()),
-                ).get_columns_from_table(model, schema)
-            except OperationalError as e:
-                raise RuntimeError(
-                    f"Could not connect to your postgres database, check your profiles.yml\n{e}"
-                )
-        elif type_of_connection == "snowflake":
-            columns_sql = SnowflakeConnector(
-                user=dbt_credentials.get("user", str()),
-                password=dbt_credentials.get("password", str()),
-                account=dbt_credentials.get("account", str()),
-                database=dbt_credentials.get("database", str()),
-            ).get_columns_from_table(model, schema)
+        connector = DB_CONNECTORS.get(type_of_connection)
+        if not connector:
+            print("The type of connector doesn't exists.")
```

Comment:
could we raise instead of printing here?

Reply:
yes, that will be better